framework/src/modules/auth/module.ts

```diff
@@ -33,10 +43,144 @@ export class AuthModule extends BaseModule {
 	public configSchema = configSchema;
 	public commands = [new RegisterMultisignatureCommand(this.id)];
 
-	// eslint-disable-next-line @typescript-eslint/require-await
-	public async verifyTransaction(_context: TransactionVerifyContext): Promise<VerificationResult> {
+	public async verifyTransaction(context: TransactionVerifyContext): Promise<VerificationResult> {
```

Comment:
it might be easier to read if we split each cases in private function?
- registerMultisignatureGroup
- multi-signature
- single-signature

Reply:
extracted to util funcs